crates/weaver_emit/src/lib.rs

```diff
@@ -113,7 +157,24 @@ pub fn emit(
                 error: e.to_string(),
             })?;
 
-        // TODO Emit metrics
+        // Emit metrics
+        let meter_provider = match tracer_provider_config {
```

Comment:
`tracer_provider_config` could be renamed/generalized into `signal_provider_config`